docs/modules/ROOT/pages/reference/extensions/joor.adoc

```diff
@@ -39,3 +43,39 @@ Please refer to the above link for usage and configuration details.
 ifeval::[{doc-show-user-guide-link} == true]
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 endif::[]
+
+[id="extensions-joor-additional-camel-quarkus-configuration"]
+== Additional Camel Quarkus configuration
+
+The expressions and scripts written using the jOOR language are extracted and compiled at build time, to do so the
+language needs to be instantiated and configured at build time in order to be able to generate the exact same source code as
+the one generated at runtime. For this purpose, the next configuration properties have been added:
+
+
+[width="100%",cols="80,5,15",options="header"]
+|===
+| Configuration property | Type | Default
+
+
+|icon:lock[title=Fixed at build time] [[quarkus.camel.joor.single-quotes]]`link:#quarkus.camel.joor.single-quotes[quarkus.camel.joor.single-quotes]`
+
+Indicates whether a jOOR expression can use single quotes instead of double quotes.
+| `boolean`
+| `true`
+
+|icon:lock[title=Fixed at build time] [[quarkus.camel.joor.config-resource]]`link:#quarkus.camel.joor.config-resource[quarkus.camel.joor.config-resource]`
```

Comment:
Just an observation - The config descriptions for `config-resource` & `result-type` don't seem to be in the doc.

Reply:
Nice catch 👍